packages/strapi-plugin-graphql/services/type-definitions.js

```diff
@@ -64,6 +64,19 @@ const buildTypeDefObj = model => {
       delete typeDef[association.alias];
     });
 
+  // Remove private attributes (which are not attributes) defined per model or globally
+  const privateAttributes = _.union(
+    _.get(model, 'options.privateAttributes', []),
+    _.get(model, 'options.ignoreGlobalPrivateAttributes', false)
```

Comment:
You forgot to refactor this

Reply:
Totally forgot about GraphQL. Will do now.